### DISCUSSION TITLE
Does it works on MHI2_US_POG11_P3300 0809?

### DISCUSSION THREAD

**Discussion by alantian88** [harman-f/MHI2_US_POG11_K5186_1-MU1476-AIO#2] [Q&A]
Hi, my car is a 2017 Porsche Cayenne running Software Version: MHI2_US_POG11_P3300 0809. Hardware Version: 051. Does the update work for my car? (The main reason for me to update is to activate Apple Carplay)

**Comment by harman-f** [comment]
Yes, it is working for all POG11 FW versions.

**Reply by Mjack74** [reply]
Sorry to revive old thread....I also have a 2017 Porsche although 911 CS4 with MHI2_US_POG11_P3300 0809. Hardware version 50. I am trying to update my maps to later version they are currently NAR 2016/2017. Can you tell me why I am getting the following error when using P336_N60S5MIBH3_NAR_2023-2024? --> "The medium contains no valid update". Others on the rennlist forum seem to be able to load via the standard menus (albeit with different Software versions) yet I am getting the error. What am I doing wrong? Do I need to install this first or is there another set of later maps I can use if all I want to do is update my Nav maps?
